hi-ml-histopathology/testhisto/testhisto/mocks/tiles_generator.py

```diff
@@ -95,7 +95,7 @@ def create_mock_metadata_dataframe(self) -> pd.DataFrame:
                 mock_metadata[self.GLEASON_SCORE].append(gleason_score)
 
         df = pd.DataFrame(data=mock_metadata)
-        df.to_csv(os.path.join(self.tmp_path, PandaTilesDataset.DEFAULT_CSV_FILENAME), index=False)
+        df.to_csv(os.path.join(self.dest_data_path, PandaTilesDataset.DEFAULT_CSV_FILENAME), index=False)
```

Comment:
Since self.dest_data_path is a Path you can use `self.dest_data_path / PandaTilesDataset.DEFAULT_CSV_FILENAME` here. I also think it's better to create a variable for this value to make it clear what it represents